api-reference/plans/object.mdx

```diff
@@ -433,12 +437,18 @@ description: "This object represents a plan. This plan can then be assigned to a
             <ResponseField name="fixed_amount" type="string">
               The fixed fee that is applied to each transaction for a `percentage` charge model. It is expressed as a decimal value.
             </ResponseField>
-            <ResponseField name="free_units_per_events" type="integer" required>
+            <ResponseField name="free_units_per_events" type="integer">
               The count of transactions that are not impacted by the `percentage` rate and fixed fee in a percentage charge model. This field indicates the number of transactions that are exempt from the calculation of charges based on the specified percentage rate and fixed fee.
             </ResponseField>
-            <ResponseField name="free_units_per_total_aggregation" type="integer" required>
+            <ResponseField name="free_units_per_total_aggregation" type="integer">
               The transaction amount that is not impacted by the `percentage` rate and fixed fee in a percentage charge model. This field indicates the portion of the transaction amount that is exempt from the calculation of charges based on the specified percentage rate and fixed fee.
             </ResponseField>
+            <ResponseField name="per_transaction_min_amount" type="string">
+              Specifies the minimum allowable spending for a single transaction, defined in cents. Working as a transaction floor. Cannot be greater than the `per_transaction_max_amount` value.
```

Comment:
`defined in cents`. Actually, for charges, amounts are not defined in cents but in "plain" amount.